CHANGELOG.md

```diff
@@ -2,15 +2,18 @@
 
 This file is used to list changes made in each version of the Java cookbook.
 
-## Unreleased
+## 8.0.0 (tbc)
+
+- Added `openjdk_install` resource & documentation
+- Deprecated openjdk recipe
```

Comment:
We should just say remove since deprecate means it works, but it's on its way out.